package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Oran.js JavaScript by Orange Cats",
   "main": "index.js",
   "scripts": {
-	 "pretest": "npm run lint",
-    "test": "gulp test",
+    "test": "eslint . --ext .js && gulp test",
+    "start": "node app/index.js",
```

Comment:
Yes it should, I honestly didn't even check that.  I didn't add that but I can take care of it.